src/Symfony/Component/Translation/Bridge/Crowdin/CrowdinProvider.php

```diff
@@ -109,6 +109,25 @@ public function read(array $domains, array $locales): TranslatorBag
         $translatorBag = new TranslatorBag();
         $responses = [];
 
+        $localeLanguageMap = [];
+        foreach ($this->listLanguages($locales) as $language) {
```

Comment:
this method doesn't accept any arguments

Reply:
Probably it would be better to extract this to a separate method, e.g. `mapLocaleToLanguageId` or something like this